courses/views.py

```diff
@@ -38,6 +41,44 @@ class ProgramViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = ProgramSerializer
 
 
+class ProgramLearnersView(APIView):
```

Comment:
could you use the `ListAPIView` class from django rest framework here? I was able to use it when I made the `AutomaticEmailView`, and it sort of did some stuff automatically for me. I'm not so sure of the API stuff so I don't know if it would work, but it might make it easier!